app/Foliage/Pages.hs

```diff
@@ -116,13 +117,16 @@ makeAllPackageVersionsPage currentTime outputDir packageVersions =
     entries =
       -- collect all cabal file revisions including the original cabal file
       foldMap
-        ( \PreparedPackageVersion {pkgId, pkgTimestamp, pkgVersionSource, cabalFileRevisions} ->
+        ( \PreparedPackageVersion {pkgId, pkgTimestamp, pkgVersionSource, pkgVersionIsDeprecated, cabalFileRevisions} ->
             -- original cabal file
             AllPackageVersionsPageEntryPackage
               { allPackageVersionsPageEntryPkgId = pkgId,
                 allPackageVersionsPageEntryTimestamp = fromMaybe currentTime pkgTimestamp,
                 allPackageVersionsPageEntryTimestampPosix = utcTimeToPOSIXSeconds (fromMaybe currentTime pkgTimestamp),
-                allPackageVersionsPageEntrySource = pkgVersionSource
+                allPackageVersionsPageEntrySource = pkgVersionSource,
+                -- FIXME: this weirdly seems to not work (display a `Deprecated` badge on all package version page) ...
```

Comment:
going to fix it now or not? It potentially suggests a bug in how we're deciding if things are deprecated or not, which seems serious...

Reply:
if only we had tests :)

Reply:
Right, I forgot about that note. It was put there before lots of changes so I wonder it's still true.